packages/gatsby/src/commands/develop-static.ts

```diff
@@ -0,0 +1,27 @@
+import sysPath from "path"
+import express from "express"
+import { ServeStaticOptions } from "serve-static"
+import parseUrl from "parseurl"
+
+export function developStatic(
+  root: string,
+  options: ServeStaticOptions
+): express.Handler {
```

Comment:
```suggestion
export const developStatic = (
  root: string,
  options: ServeStaticOptions
): express.Handler => {
```

Reply:
We don't have guidelines about this. If we had guidelines like that we would also have lint rules for it. So I'll merge this without proposed changes